src/tools/store.cpp

```diff
@@ -18,10 +17,7 @@ bool generateDataStoreOptions(const int argc,
     // declare a group of options that will be allowed only on command line
     boost::program_options::options_description generic_options("Options");
     generic_options.add_options()("version,v", "Show version")("help,h", "Show this help message")(
-        "springclean,s", "Remove all regions in shared memory")(
-        "config,c", boost::program_options::value<boost::filesystem::path>(&paths["config"])
-                        ->default_value("server.ini"),
-        "Path to a configuration file");
+        "springclean,s", "Remove all regions in shared memory");
```

Comment:
If you `clang-format` and want to force the options to appear on a new line, you can do it like this:

    add_option()                     //
      ("version", "Show version")    //
      ("help", "Show help");         //